packages/wp-now/src/execute-php.ts

```diff
@@ -21,14 +23,21 @@ export async function executePHP(
 		);
 	}
 	disableOutput();
-	const { phpInstances } = await startWPNow({
+	const { phpInstances, options: wpNowOptions } = await startWPNow({
 		...options,
 		numberOfPhpInstances: 2,
 	});
 	const [, php] = phpInstances;
 
 	try {
 		php.useHostFilesystem();
+		const maybePhpFile = path.join(
```

Comment:
It fixes the issue, but I don't like that implementation. Any thoughts on how to make it better?

Reply:
Yeah, I don't love it either.

I think the user's expectation is that the provided file path is relative to their current working directory. Can we prepend the current working directory if the file path isn't absolute?

Reply:
@danielbachhuber I've added a check for the absolute path, but we still need to check if the path exists as with the current implementation. The reason is that this parameter can be `-v` for the `wp-now php -v` call or `--path=/Sites/my-site` for the `wp-now php` call. 

I've added https://github.com/WordPress/playground-tools/issues/76 to improve php command parameters usage.